Sources/MapboxDirectionsCLI/CodingOperation.swift

```diff
@@ -57,8 +107,14 @@ class CodingOperation<ResponceType : Codable, OptionsType : DirectionsOptions >
         
         decoder.userInfo = [.options: directionsOptions,
                             .credentials: BogusCredentials]
+        
+        var routeResponse: RouteResponse?
+        if options.outputFormat == .gpx {
+            let gpxData = try String(contentsOfFile: options.inputPath).data(using: .utf8)!
+            routeResponse = try! decoder.decode(RouteResponse.self, from: gpxData)
```

Comment:
```suggestion
            routeResponse = try! decoder.decode(RouteResponse.self, from: input)
```
And what if we do this within the `case .gpx:` in `processOutput` method using the existing `data`? Is it possible?

Reply:
yep. included instead. with previous iteration, this wasn't possible.